connectors/agentic-ai/src/main/java/io/camunda/connector/agenticai/aiagent/model/JobWorkerAgentResponse.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.agenticai.aiagent.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.camunda.connector.agenticai.model.AgenticAiRecord;
+import io.camunda.connector.agenticai.model.message.AssistantMessage;
+import org.springframework.lang.Nullable;
+
+@AgenticAiRecord
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public record JobWorkerAgentResponse(
```

Comment:
:question: don't we need a `@JsonDeserialize` like in `AgentResponse`?

Reply:
Good catch! The main advantage of forcing the deserialization through the builders is to leverage features like the implicit null checks and that collections are always initialized vs. being null.

As this record has all nullable fields and no collections, it's not strictly necessary, but still I'll add it for consistency and for potential future changes.